pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1502,6 +1502,32 @@ bool GSQLBackend::replaceComments(const uint32_t domain_id, const string& qname,
   return true;
 }
 
+string GSQLBackend::directGSQLQuery(const string &query)
+{
+ try {
+   ostringstream out;
+
+   unique_ptr<SSqlStatement> stmt(d_db->prepare(query,0));
```

Comment:
I checked goracle, gmysql, gpgsql and gsqlite3 and they all perform cleanup in dtor.